yarn-project/p2p/src/bootstrap/bootstrap.ts

```diff
@@ -72,16 +67,29 @@ export class BootstrapNode implements P2PBootstrapApi {
     });
 
     try {
+      this.logger.info('Starting bootnode');
       await this.node.start();
       this.logger.info('Bootstrap node started', {
         peerId,
         enr: enr.encodeTxt(),
         addr: listenAddrUdp.toString(),
-        versions,
       });
     } catch (e) {
       this.logger.error('Error starting Discv5', e);
     }
+
+    // Add bootnode ENRs if provided, making sure we filter our own
```

Comment:
Heads up we are not filtering our own here

Reply:
Done